Danil_Safronov/GC.scala

```diff
@@ -0,0 +1,23 @@
+import scala.io.StdIn.readLine
+import DNA.count
+
+import scala.io.Source
+
+object GC {
```

Comment:
Garbage Collector?

Reply:
The GC-content of a strand of nucleic acid is the percentage of nucleotides in the strand that possess either cytosine or guanine bases.  :-)